firebase-firestore/src/test/java/com/google/firebase/firestore/local/SQLiteIndexManagerTest.java

```diff
@@ -1189,6 +1190,49 @@ public void testIndexTypeForOrQueries() throws Exception {
     validateIndexType(query11, IndexManager.IndexType.PARTIAL);
   }
 
+  @Test
+  public void TestCreateTargetIndexesCreatesFullIndexesForEachSubTargets() {
```

Comment:
nit: `TestCreateTargetIndexesCreatesFullIndexesForEachSubTargets` -> `TestCreateTargetIndexesCreatesFullIndexesForEachSubTarget` (that is, remove the trailing `s` on `Targets`).